src/EditorFeatures/CSharp/EditorConfigSettings/DataProvider/Formatting/CSharpFormattingSettingsProvider.cs

```diff
@@ -74,6 +74,7 @@ private static IEnumerable<FormattingSetting> GetNewLineOptions(AnalyzerConfigOp
             yield return FormattingSetting.Create(CSharpFormattingOptions2.NewLinesForBracesInMethods, CSharpEditorResources.Place_open_brace_on_new_line_for_methods_local_functions, editorConfigOptions, visualStudioOptions, updaterService);
             yield return FormattingSetting.Create(CSharpFormattingOptions2.NewLinesForBracesInProperties, CSharpEditorResources.Place_open_brace_on_new_line_for_properties_indexers_and_events, editorConfigOptions, visualStudioOptions, updaterService);
             yield return FormattingSetting.Create(CSharpFormattingOptions2.NewLinesForBracesInAccessors, CSharpEditorResources.Place_open_brace_on_new_line_for_property_indexer_and_event_accessors, editorConfigOptions, visualStudioOptions, updaterService);
+            yield return FormattingSetting.Create(CSharpFormattingOptions2.NewLinesForBracesInPropertyPatternClauses, CSharpEditorResources.Place_open_brace_on_new_line_for_property_pattern_clauses, editorConfigOptions, visualStudioOptions, updaterService);
```

Comment:
❗ This should reuse the existing option `NewLinesForBracesInObjectCollectionArrayInitializers`, and not add a new option. The design that established this precedent (which it's certainly understandable you didn't see because it's a comment buried in a seemingly-unrelated topic) is https://github.com/dotnet/roslyn/issues/36913#issuecomment-546015089.

Reply:
I see, makes sense if it's getting renamed to `NewLinesForBracesInExpressions` eventually. Will do.

Reply:
@mrahhal I don't think renaming it is desired. It's a [public shipped API](https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.formatting.csharpformattingoptions.newlinesforbracesinobjectcollectionarrayinitializers?view=roslyn-dotnet-3.9.0#Microsoft_CodeAnalysis_CSharp_Formatting_CSharpFormattingOptions_NewLinesForBracesInObjectCollectionArrayInitializers), so renaming will be a breaking change.

Reply:
I meant will revert to reusing `NewLinesForBracesInObjectCollectionArrayInitializers`. I won't be renaming this!